GUI/QtQuick/qml/WalletProperties/ChangePassword.qml

```diff
@@ -9,13 +9,30 @@ import "../StyledControls"
 ColumnLayout {
     id: layout
 
+    signal sig_success()
+
     height: 548
     width: 580
 
     spacing: 0
 
     property var wallet_properties_vm
 
+    Connections
+    {
+        target:bsApp
+        function onSuccessChangePassword ()
+        {
+            layout.sig_success()
+        }
+    }
+
+    CustomMessageDialog {
+        id: error_dialog
+        error: qsTr("Password should be over 6 charaters")
```

Comment:
typo (charaters)
Also "6" should be taken from the constant added earlier to QtQuickAdapter.cpp, or change the text to something like "password strength is insufficient" or "password failed to pass validation checks".

Reply:
bsApp.validatePassword is used in password integrity check, so I will use "password strength is insufficient" as error message.